example/with_extra/main.go

```diff
@@ -80,9 +80,9 @@ func main() {
 		},
 		Transport: &devNullTransport{},
 
-		// Solution 2 (use custom integration, which will also be
-		// applied to all events, but can be easily extracted even as a
-		// separate utility and reused across all your projects):
+		// Solution 2 (use custom integration, which will be
+		// applied to all events, and can be extracted as a
+		// separate utility and reused across projects):
```

Comment:
nit: Think this flows better

```suggestion
		// Solution 2 (use custom integration, which will be
		// applied to all events, can be extracted as a
		// separate utility, and reused across projects):
```